### PR TITLE
cakephp 4.x version

### PR DESCRIPTION
**Pull request by skie**

### CODE REVIEW

File: config/bootstrap.php [hunk 1]
```diff
@@ -5,6 +5,6 @@
 use Cake\Routing\DispatcherFactory;
 
 $appClass = Configure::read('App.namespace') . '\Application';
-if (!class_exists($appClass)) {
-    deprecationWarning('You need to upgrade to the new Http Libraries to use AssetCompress.');
-}
+// if (!class_exists($appClass)) {
+    // deprecationWarning('You need to upgrade to the new Http Libraries to use AssetCompress.');
+// }
```
[the remark this answers is not in the document]
Comment (skie) [reply config/bootstrap.php]: sure

**Comment by skie** [comment]
Why this project disabled in travis?

**Comment by markstory** [comment]
@skie Probably because I accidentally logged into travis-ci.com one time. I'll get that fixed.

**Comment by markstory** [comment]
Thank you :clap:

**Pull request merged**